dashboard/deploy_dashboard_gcloud

```diff
@@ -11,6 +11,30 @@ cd $ROOT/dashboard
 PROJECT=$1
 shift
 
+udmi_state=udmi_state
```

Comment:
what are these variables used for?

Reply:
This sets defaults for the different functions